app/models/miq_provision_virt_workflow.rb

```diff
@@ -269,6 +269,8 @@ def update_field_visibility(options = {})
 
     update_field_visibility_linked_clone(options, f)
 
+    update_field_required(f)
```

Comment:
Since the method is no longer updating the required property directly I would pick another name.  Maybe `update_field_visibility_pxe_iso` to match the line above.